structuredheadings/dialogs/selectstyle.js

```diff
@@ -1,13 +1,106 @@
+var selectedStyle;
+
+var columns = 4;
+var htmlContent = "";
+htmlContent += "<div>";
+htmlContent += "<table role=\"listbox\"";
+htmlContent += "style=\"width:100%;height:100%;border-collapse:separate;\"";
+htmlContent += "cellspacing=\"2\" cellpadding=\"2\"";
+if (CKEDITOR.env.ie && CKEDITOR.env.quirks) {
+  htmlContent += "style=\"position:absolute;\"";
+}
+htmlContent += "><tbody>";
```

Comment:
Hmm.... As far as I can tell the HTML content is required to be passed as one string, but that does sound like it might work. However I just spent a couple days making my eyes bleed sticking these strings together so it'l have to come as an update. :)